lib/simulation/openfisca/mapping/common.js

```diff
@@ -0,0 +1,15 @@
+var _ = require('lodash');
+var moment = require('moment');
+
+var isIndividuValid = exports.isIndividuValid = function(individu, situation) {
```

Comment:
Rather declare with
```js
exports.isIndividuValid = isIndividuValid;
function isIndividuValid(…)`
```

You could also group `exports` assignments, either at the end or beginning. But avoid this double assignment, it is uncommon and makes it too easy to miss the export. Plus, it's inconsistent within the file.